shared/people/container.js

```diff
@@ -39,9 +66,9 @@ const mapDispatchToProps = (dispatch, {navigateAppend}) => ({
 
 const mergeProps = (stateProps, dispatchProps) => {
   return {
-    newItems: stateProps._newItems.toJS(),
-    oldItems: stateProps._oldItems.toJS(),
-    followSuggestions: stateProps.followSuggestions.toJS(),
+    newItems: stateProps._newItems,
+    oldItems: stateProps._oldItems,
```

Comment:
we should get rid of these underscored names

Reply:
done